CONTRIBUTING.md

```diff
@@ -11,6 +11,8 @@ We welcome suggested improvements and bug fixes for `sentry-cocoa`, in the form
 
 ## Setting up an Environment
 
+💡 You might be able to do everything described below by running `make init`.
```

Comment:
`m`: I think we should be more confident here. Why not just write here that you can use `make init` or follow the steps below?

Reply:
absolutely, will update this section!

Reply:
Please check out my changes in https://github.com/getsentry/sentry-cocoa/pull/1536/commits/97714f0e216597b8c97a5bb013fba4ecc45bfb1c